libraries/AP_Torqeedo/AP_Torqeedo.h

```diff
@@ -0,0 +1,111 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+ 
+#pragma once
+
+#include <AP_HAL/AP_HAL.h>
+
+#ifndef TORQEEDO_ENABLED
+#define TORQEEDO_ENABLED !HAL_MINIMIZE_FEATURES
```

Comment:
Please check flash size rather than `HAL_MINIMIZE_FEATURES` (which we want to kill)

Reply:
Really?  I was not aware that was the plan... in any case, I've changed this to check "BOARD_FLASH_SIZE > 1024"